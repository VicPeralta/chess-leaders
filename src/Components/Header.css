
.header {
  background-color: var(--highlighted-card);
  width: 90%;
  padding: 10px 10px;
  margin: 5px auto;
  display: flex;
  justify-content: space-around;
}

.header h1 {
  margin: 0;
}

.header img {
  width: 80px;
  height: auto;
}
