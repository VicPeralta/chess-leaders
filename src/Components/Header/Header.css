
.header {
  background-color: var(--highlighted-card);
  width: 90%;
  padding: 10px 10px;
  margin: 5px auto;
  display: flex;
  justify-content: space-around;
  align-items: center;
}

.header h1 {
  margin: 0;
}

.header img {
  width: 80px;
  height: auto;
}

@media (min-width: 513px) {
  .header {
    justify-content: center;
    gap: 50px;
  }
}
