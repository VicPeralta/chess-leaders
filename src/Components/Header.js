import './Header.css';
import king from '../assets/whiteQueen.png';

const Header = () => (
  <div className="header">
    <img src={king} alt="Header" />
    <h1>
      chess.com
      <br />
      leaders
    </h1>
  </div>
);

export default Header;
