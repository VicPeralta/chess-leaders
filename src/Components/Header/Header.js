import king from '../../assets/whiteQueen.png';
import './Header.css';

const Header = () => (
  <div className="header">
    <img src={king} alt="Header" />
    <h1>
      chess.com
      <br />
      leaders
    </h1>
  </div>
);

export default Header;
