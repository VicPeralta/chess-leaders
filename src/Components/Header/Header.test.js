import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Header from './Header';

describe('Header component test', () => {
  test('Header rendering', () => {
    render(<Header />);
    const chessTitle = screen.getByText(/chess.com/i);
    expect(chessTitle).toBeInTheDocument();
  });

  test('Header snapshot', () => {
    const header = renderer.create(<Header />);
    expect(header).toMatchSnapshot();
  });
});
