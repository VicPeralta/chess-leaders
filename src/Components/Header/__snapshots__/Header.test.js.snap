// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test Header snapshot 1`] = `
<div
  className="header"
>
  <img
    alt="Header"
    src="whiteQueen.png"
  />
  <h1>
    chess.com
    <br />
    leaders
  </h1>
</div>
`;
